cmd/list_bundles_test.go

```diff
@@ -1,15 +1,48 @@
 package cmd
 
 import (
+	"log"
+	"os"
+	"os/exec"
+	"path/filepath"
+
 	. "github.com/onsi/ginkgo/v2/dsl/core"
 	. "github.com/onsi/gomega"
 )
 
 var _ = Describe("List Bundles Cmd", func() {
 	When("Calling opcap list bundles", func() {
+		// create temporary directory for testrepo
+		var tmpDir string
+		var gitUrl string
+		var err error
+
+		BeforeEach(func() {
+			tmpDir, err = os.MkdirTemp("../internal/bundle/", "testrepo")
```

Comment:
```suggestion
			tmpDir, err = os.MkdirTemp("", "testrepo-*")
```

The '*' is important... It's a placeholder that will have a generated name, ensuring that two tests get different dirs.

Also, is there a particular reason to use "../internal/bundle/" as the base directory for the temp? I would rather leave that as "" so that the system temp dir is used.